src/configure-webpack/index.js

```diff
@@ -106,7 +99,7 @@ const buildSharedWebpackConfig = (saguiConfig) => {
       // Should first try to resolve loaders nested within Sagui.
       // This fixes an issue in NPM v2 where webpack incorrectly
       // thinks that the package `eslint` is the `eslint-loader`
-      modulesDirectories: [
```

Comment:
The comments points out to an npm 2 problem, but I imagine npm 2 is _super deprecated_ at this point. Is this in general necessary?